src/common/components/Tooltip/Skin/styles.less

```diff
@@ -0,0 +1,30 @@
+@import '~common/styles/base.less';
+
+.description {
+  display: flex;
+  align-items: flex-start;
+}
+
+.detailsIcon {
+  margin-right: 5px;
+}
+
+.tooltipIcon {
+  border-radius: 1px;
+  border-left: 1px solid @_gray;
+  border-top: 1px solid @_gray;
+  border-right: 1px solid @_darkergray;
+  border-bottom: 1px solid @_darkergray;
+  float: left;
+  margin-top: 3px;
+  margin-right: 5px;
+  vertical-align: middle;
+}
+
+.skinNameText {
```

Comment:
looks like this is the only style needed now right? we can get rid of everything else :D